src/canvas-extractor/edfi_canvas_extractor/extract_graphql.py

```diff
@@ -75,10 +109,13 @@ def run(arguments: MainArguments) -> None:
 
     for account in accounts:
         _id = getattr(account, "id")
-        gql = Extract()
-        gql.set_credentials(arguments.base_url, arguments.access_token)
-        gql.set_account(_id)
-        gql.set_dates(arguments.start_date, arguments.end_date)
+        gql = GraphQLExtractor(
+            arguments.base_url,
+            arguments.access_token,
+            _id,
+            arguments.start_date,
+            arguments.end_date
+            )
         gql.run()
```

Comment:
We are not getting any files for account_id = 2. There is a scope problem: note that `gql` is instantiated on line 112, inside the `for` loop. However, we then use `gql` _outside_ the for loop, starting at line 124. Thus we end up only getting the data from the last account in the list from `get_accounts`.

Reply:
applied changes